### PR TITLE
add day-to-day category

### PR DESCRIPTION
**Pull request by davidthomas22**

### CODE REVIEW

**Comment by davidthomas22** [comment]
added new category and updated CSS and test files

File: moj-node/assets/sass/components/_header.scss [hunk 1]
```diff
@@ -91,6 +91,10 @@
         background: govuk-colour("red");
     }
 
+    &.Day-to-day {
```
Comment (romidane) [inline moj-node/assets/sass/components/_header.scss]: why is this capitalised?

File: moj-node/server/views/components/landing-page-header/template.njk [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 {% set categoryName = {
   "644": "news-events",
+  "787": "Day-to-day",
```
Comment (romidane) [inline moj-node/server/views/components/landing-page-header/template.njk]: why is this capitalised `Day`?

Comment (davidthomas22) [reply moj-node/server/views/components/landing-page-header/template.njk]: good call

**Pull request merged**